### ISSUE TITLE
Loading tracks with decorations from the track menu crashes client.

### ISSUE THREAD

**Issue by Nubebuster** [OpenRCT2/OpenRCT2#3616]
**OS:** Windows 10 upgraded from Windows 7
**Version:** 0.0.5.0
**Commit:** 6f7db1f

Explanation of the issue...
I started a game in multiplayer of the map 'Bumbly Bazaar'. Then I saved it and went to singleplayer to add prebuild tracks. I added chocolate log (with no decorations) and that worked. Then I went to add Mine Mania (with decorations) and as soon as I click on the button in the prebuilt tracks selection, the game crashes. I have done this twice, once with Mine Mania and once with any other track.
- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Multiplayer? (not applicable)

**Steps to reproduce:**
(The first two steps might not be necessary)
1. Load a game in multiplayer
2. Save that game
3. Load the save in singleplayer
~~4. Load any track that has decorations~~
4. Load Mine Mania or King Rapids (not all tracks with decorations cause this)

**Dump file**
If you have a dump file, zip it and drag&drop it here.

[OpenRCT2 bug report.zip](https://github.com/OpenRCT2/OpenRCT2/files/265060/OpenRCT2.bug.report.zip)

**Screenshots / Video:**
Drag / drop screenshots here. Use https://vid.me to upload video.

**Save game:**
Change the file extension to .txt or package to a .zip so that it can be drag / dropped here...

**Comment by janisozaur** [comment]
1. Please format your reports properly, remove the parts you don't need from the template, as otherwise it's hard for us to read
2. This a bug report for an old version, can you check if it happens in current builds?

**Comment by Nubebuster** [comment]
I downloaded the latest version. How do I get the latest latest version?

**Comment by janisozaur** [comment]
https://openrct2.org/downloads

**Comment by Nubebuster** [comment]
I actually made a mistake, I was running the latest version.

**Comment by Nubbie** [comment]
OPs dumpfile 86c29a6f-4259-495f-a609-249b12857b61(3a6978a).dmp

Error message;
Unhandled exception thrown: read access violation.
_loadedTrackDesign was nullptr.

Call Stack;
```
>	openrct2.dll!window_track_list_select(rct_window * w=0x103e4120, int index) Line 200	C
 	openrct2.dll!input_scroll_begin(rct_window * w=0x103e4120, int widgetIndex=3, int x, int y) Line 602	C
 	openrct2.dll!input_widget_left(int x=46, int y=77, rct_window * w=0x103e4120, int widgetIndex=3) Line 1027	C
 	openrct2.dll!game_handle_input_mouse(int x=46, int y=77, int state=1) Line 312	C
 	openrct2.dll!game_handle_input(...) Line 151	C
 	openrct2.dll!game_update(...) Line 344	C
 	openrct2.dll!rct2_update(...) Line 467	C
 	openrct2.dll!openrct2_loop(...) Line 456	C
 	openrct2.dll!openrct2_launch(...) Line 324	C
 	openrct2.dll!StartOpenRCT(HINSTANCE__ * hInstance, HINSTANCE__ * hPrevInstance=0x00000000, char * lpCmdLine=0x016a229f, int nCmdShow=1) Line 110	C
 	openrct2.exe!004069e4()	Unknown
 	[Frames below may be incorrect and/or missing, no symbols loaded for openrct2.exe]	
 	[External Code]	
```

Autos;
```
		gTrackDesignSceneryToggle	false	bool
```

Locals;
```
		index	Variable is optimized away and not available.	
-		tdRef	0x05f28ad0 {name=??? path=??? }	track_design_file_ref *
		name	<Unable to read memory>	
		path	<Unable to read memory>	
+		w	openrct2.dll!0x103e4120 {event_handlers=0x1035cf50 {openrct2.dll!rct_window_event_list window_track_list_events} {...} ...}	rct_window *
```

**Comment by janisozaur** [comment]
Duplicate of https://github.com/OpenRCT2/OpenRCT2/issues/3623, fixed in 52a92fa
